.github/workflows/dev.yml

```diff
@@ -270,22 +276,72 @@ jobs:
         with:
           limit-access-to-actor: true
 
+  vlab:
+    if: "${{ needs.check_changes.outputs.devfiles == 'true' }}"
+    needs:
+      - check
+
+    name: "${{ matrix.hybrid && 'hlab' || 'vlab' }}-${{ matrix.fabricmode == 'spine-leaf' && 'sl' || 'cc' }}-${{ matrix.gateway && 'gw-' || '' }}${{ matrix.includeonie && 'onie-' || '' }}${{ matrix.buildmode }}-${{ matrix.vpcmode }}"
+
+    uses: githedgehog/fabricator/.github/workflows/run-vlab.yaml@master
+    with:
+      skip: ${{ matrix.hybrid && !contains(github.event.pull_request.labels.*.name, 'ci:+hlab') || !matrix.hybrid && !contains(github.event.pull_request.labels.*.name, 'ci:+vlab') }}
+      fabricatorref: master
```

Comment:
Should there be a way to configure this to pass a specific fabricator reference?

Reply:
Yes, that's the use cases I had in mind :slightly_smiling_face:

Reply:
OK I can work with that, indeed